scipy/sparse/csgraph/bento.info

```diff
@@ -5,6 +5,8 @@ Library:
         Sources: _traversal.c
     Extension: _min_spanning_tree
         Sources: _min_spanning_tree.c
+    Extension: _max_flow
+        Sources: _max_flow.c
```

Comment:
oops, this is a leftover file from a build system we no longer support. cleaned up now

Reply:
Gotcha; merged the change into this one and fixed the documentation issue.